frontend/src/metabase/internal/pages/WelcomePage.jsx

```diff
@@ -1,18 +1,35 @@
 import React from "react";
+import { Box } from "grid-styled";
 
 import Heading from "metabase/components/type/Heading";
+import Subhead from "metabase/components/type/Subhead";
 import Text from "metabase/components/type/Text";
 
+import ProgressBar from "metabase/components/ProgressBar";
+
+import { stats } from "../lib/components-webpack";
+
 const WelcomePage = () => {
   return (
-    <div className="wrapper wrapper--trim">
-      <div className="my4">
+    <Box className="wrapper wrapper--trim">
+      <Box my={4}>
         <Heading>Metabase Style Guide</Heading>
         <Text>
           Reference and samples for how to make things the Metabase way.
         </Text>
-      </div>
-    </div>
+      </Box>
+
+      <Subhead>Documentation progress</Subhead>
+      <Text>
+        Documenting our component library is an ongoing process. Here's what
+        percentage of the code in <code>metabase/components</code> has some form
+        of documentation so far.
+      </Text>
+
+      <Box mt={3}>
+        <ProgressBar percentage={stats.ratio * 0.01} />
```

Comment:
It seems like this should be `ratio * 100` since `ratio` will be in [0,1] and `ProgressBar` seems to expect something in `[0,100]`.

It looks like it's working in the screenshot, so I guess I'm missing something?

Reply:
Oops I had missed that the `* 100` already happens in `ProgressBar`.